losses/binary_losses.py

```diff
@@ -44,7 +45,7 @@ def convert_to_logits(y_pred: tf.Tensor) -> tf.Tensor:
     # To avoid unwanted behaviour of log operation
     y_pred = K.clip(y_pred, tf.keras.backend.epsilon(), 1 - tf.keras.backend.epsilon())
 
-    return K.log(y_pred / (1 - y_pred))
+    return K.log(y_pred / (1 - y_pred) + epsilon)
```

Comment:
Is this epsilon really necessary? What I can tell, the clipping in the line above should handle so that `y_pred != 0` and `y_pred != 1`